lib/cereal/postgres/sql_schema.go

```diff
@@ -196,6 +198,7 @@ AS $$
         VALUES('cancel', _workflow_instance_id);
 $$ LANGUAGE SQL;
 
+-- TODO(jaym): This function looks wrong now
```

Comment:
How so?  I'm thinking maybe it was wrong in a previous iteration but should be correct again now that we've reduced this down to just pining updated_at?

Reply:
i've basically forgotten why we had abandon. I feel like it should just be complete

Reply:
the way unclaimed tasks are selected also is wrong now

Reply:
We have abandoned in case you complete a task when there are still outstanding tasks.  I agree this feels like it could just be part of complete.